private/buf/cmd/buf/command/push/push.go

```diff
@@ -346,6 +346,37 @@ func createUploadModulesIfNotExist(
 	return nil
 }
 
+func validateFlags(flags *flags) error {
+	if err := validateCreateFlags(flags); err != nil {
+		return err
+	}
+	if err := validateLabelFlags(flags); err != nil {
+		return err
+	}
+	return nil
+}
+
+func validateLabelFlags(flags *flags) error {
+	for _, label := range flags.Labels {
+		if label == "" {
+			return appcmd.NewInvalidArgumentErrorf(
+				"--%s requires a non-empty string.",
+				labelFlagName,
+			)
+		}
+	}
+	for _, tag := range flags.Tags {
+		if tag == "" {
+			return appcmd.NewInvalidArgumentErrorf(
+				"%s|--%s requires a non-empty string.",
```

Comment:
drive by, but should the short flag have a `-` in front?

```suggestion
				"-%s|--%s requires a non-empty string.",
```

Reply:
Ah I `x`'d that by accident. Let me push that up.